.travis.yml

```diff
@@ -33,11 +33,14 @@ install:
 script:
   - npm test
   - NODE_ENV=production node_modules/.bin/builder run build-static
+  - node_modules/.bin/builder run install-selenium
+  - ROWDY_SETTINGS=local.firefox node_modules/.bin/builder run test-func-static-prod
```

Comment:
Are these tasks in `package.json`? I think by default`test-func-static-prod` are not available in the archetype...